index.html

```diff
@@ -1,9 +1,106 @@
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
 <head>
-	<title></title>
+<title>Cultural Club</title>
+<style type="text/css">
+	.hit-the-floor {
+color: #fff;
+font-size: 6em;
+font-weight: bold;
+font-family: Helvetica;
+text-shadow: 0 1px 0 #ccc, 0 2px 0 #c9c9c9, 0 3px 0 #bbb, 0 4px 0 #b9b9b9, 0 5px 0 #aaa, 0 6px 1px rgba(0,0,0,.1), 0 0 5px rgba(0,0,0,.1), 0 1px 3px rgba(0,0,0,.3), 0 3px 5px rgba(0,0,0,.2), 0 5px 10px rgba(0,0,0,.25), 0 10px 10px rgba(0,0,0,.2), 0 20px 20px rgba(0,0,0,.15);
+}
+
+.hit-the-floor {
+  text-align: center;
+}
+#img1
+{
+    float: left;
+}
+#img2
+{
+    float: right;
+}
+
+</style>
 </head>
 <body>
-<h1>Indian Institute of Information Technology</h1>
+<div class="hit-the-floor">
+	<div id="img1"><img src="a.jpg" width=250px height=150px></div>
+	WILDBEATS
+	<div id="img2"><img src="a.jpg" width=250px height=150px></div>
+</div>
+<br><br>
+<div class="container" style="width:100%;">
+  <div id="myCarousel" class="carousel slide" data-ride="carousel">
+    <!-- Indicators -->
+    <ol class="carousel-indicators">
+      <li data-target="#myCarousel" data-slide-to="0"></li>
+      <li data-target="#myCarousel" data-slide-to="1" class="active"></li>
+      <li data-target="#myCarousel" data-slide-to="2"></li>
+    </ol>
+
+    <!-- Wrapper for slides -->
+    <div class="carousel-inner">
+
+    <div class="item">
+        <img alt="Diwali" style="width:100%; height:300px;">
+        <div class="carousel-caption">
+          <h3>Diwali Celebrations at IIITK</h3>
+          <p> Family Environment away from Home </p>
+        </div>
+      </div>
+
+      <div class="item active">
+        <img alt="Cultural Fest" style="width:100%; height:300px;">
+        <div class="carousel-caption">
+          <h3>Cultural Fest at IIITK</h3>
+          <p>Registrar Dr. Radhakrishnan Sir Addressing Students</p>
+        </div>
+      </div>
+    
+      <div class="item">
+        <img alt="Holi" style="width:100%; height:300px;">
+        <div class="carousel-caption">
+          <h3>Holi Vibes at IIITK</h3>
+          <p>Having fun with colours</p>
+        </div>
+      </div>
+  
+    </div>
+
+    <!-- Left and right controls -->
+    <a class="left carousel-control" href="#myCarousel" data-slide="prev">
+      <span class="glyphicon glyphicon-chevron-left"></span>
+      <span class="sr-only">Previous</span>
+    </a>
+    <a class="right carousel-control" href="#myCarousel" data-slide="next">
+      <span class="glyphicon glyphicon-chevron-right"></span>
+      <span class="sr-only">Next</span>
+    </a>
+  
+  </div>
+  <br>
+  <div class="well">
+  <p style="text-align: left; font-size:2em; color: blue; font-family: Helvetica;">WildBeats ...</p>
```

Comment:
let's not do this, applying css to individual elements which repeats, would make it inconsistent in future, create appropriate class in and assign class to related elements

Reply:
I see this pattern in multiple elements, try to change this..